packages/core/data-transfer/src/strapi/queries/link.ts

```diff
@@ -338,7 +338,16 @@ const filterValidRelationalAttributes = (attributes: Record<string, any>) => {
   };
 
   const isComponentLike = (attribute: any) => {
-    return attribute.component || attribute.components;
+    return (
+      /**
+       * probably can remove attribute.component & attribute.components
+       * to be checked
+       *  */
+      attribute.component ||
+      attribute.components ||
+      attribute.joinTable?.name.includes('components') ||
```

Comment:
How is that possible to have a relation to components/cmps that is not a component/dynamic zone?

What is the shape of the attribute you get here?

Reply:
it's not different, I just think the object we are receiving doesn't contain this anymore attribute.component  or
      attribute.components

Reply:
I see, could you make it so that we're checking the suffix instead of a loose `includes`? This should prevent having false positives.

Also, you can indeed remove the previous check.

Reply:
Just curious what causes this bug on v5 but not on v4. Are we just covering up another issue?

Reply:
Basically: can you see if this issue exists or not in v4?

If the issue also exists in v4, this fix seems ok (but will also need to be merged in v4 as well).

If this issue doesn't exist in v4, it means that something unexpected has changed in the objects sent, and it could be an underlying problem we need to look into.

Reply:
it's only a v5 issue, the v4 object contains the component/components attribute, it seems it's been removed in v5
at the same time it doesn't look like it's been used anywhere else expect here previously